src/sidebar/components/test/annotation-test.js

```diff
@@ -18,6 +18,16 @@ var fakeDocumentMeta = {
 };
 
 /**
+ * Return an `Error` object matching the type of error produced by failed API
+ * calls.
+ */
+function apiError(status, message) {
```

Comment:
So this is meant to match the type or `Error` thrown by `store.js`? I wonder if this could become a constructor function in `store.js` itself, `store` could then use it when it wants to construct an error, and these tests could import and use the real error constructor function from `store`.

That would remove some duplication (between `store.js` and this test file) and remove the risk of this test helper getting out of sync with `store.js`. If I'm reading it correctly it might also make `store.js`'s `translateResponseToError()` function clearer - that function looks like it's both defining what a store error looks like, and translating an API response into a store error. An `APIError` constructor could define what a store error looks like, leaving `translateResponseToError()` to call it and only have to define how to translate API responses.